addons/main/CfgGesturesMale.hpp

```diff
@@ -1,50 +1,110 @@
 class CfgMovesBasic {
     class ManActions {
         GVAR(addPlate_base)= QGVAR(addPlate_base);
-        GVAR(addPlate_2)= QGVAR(addPlate_2);
```

Comment:
Do you know of a better way to do this?

Reply:
Not really. gestures have a set speed, you cannot speed them up with animation Coef. so you have to suffer through the config hell if you REALLY want this.

Why not just lower the head bob value and be fine with it? Would that be an option?

Reply:
That's what I was going to do, but ["Can you make it user configurable?"](https://discord.com/channels/230161070277263372/858157366590701568/985381901709742140) struck again.

Reply:
I guess the real question is whether there's anyone who actually wants the headbob. 
I'm fine with just picking a good value and leaving it at that. I'm just assuming someone is going to pop up and say "but I liked it the other way!"

Reply:
Oh also, my original question was referring to having animation versions with multiple variables without needing to explicitly specify every permutation.

Maybe some macro trickery?

Reply:
it is possible, a new macro needs to be created that is a multiline macro. one line per new gesture.. so in the end you have only one line in the hpp file but it does add multiple once compiled.